Issue 51: Updated resources page with category and language filters

aszheng opened the pull request:
- Cleaned and updated the resources Google Doc spreadsheet 
- Added filters for categories (Affordable Housing, Legal Aid, Social Service, Shelter) and languages
- Languages based on availability on website's / bilingual services provided at the physical location/phone

ychoy commented:
The filter button is difficult to see in contrast with the list of resources. I suggest putting the word Resources on top of the Filter section, and then making the filter and all the categories fixed to the top of the page underneath the Resources header instead of an accordian menu item.  

Can you change radio buttons into checkboxes? In case a user wants to select multiple categories to filter resources.

aszheng commented:
-Moved radio buttons into checkboxes
-Support for multiple categories in params
-Moved filter boxes to the left

-TODO: CSS updates for mobile support

ychoy commented:
Thanks for working on this @aszheng.
